### DISCUSSION TITLE
"Show Context" in Split-View

### DISCUSSION THREAD

**Discussion by lowb1rd** [grafana/grafana#48302] [General]
I'm using Grafana Loki. The "show context" feature is too limited.

Describe the solution you'd like
"Show Context" should open in the split view, highlighting the filtered line and showing the context of that file (say +/- 10 log lines). One could add then additional files to even see data from other logs/systems in that context. "Load more" entries should also be possible to skip further back and forth.

**Comment by hunkjun** [comment]
Like this: https://github.com/grafana/grafana/issues/50913

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
